server/svix-server/src/lib.rs

```diff
@@ -15,6 +18,7 @@ use std::{
 use tower::ServiceBuilder;
 use tower_http::cors::{AllowHeaders, Any, CorsLayer};
 use tower_http::trace::TraceLayer;
+use tracing_subscriber::{prelude::*, util::SubscriberInitExt};
```

Comment:
Why is this prelude even needed now?

Reply:
We still need `__tracing_subscriber_SubscriberExt`, for
```rust
tracing_subscriber::Registry::default()
                .with(...)
```
which comes from the prelude